src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -558,6 +569,30 @@ const getOpenscap = (state: RootState): OpenScap | undefined => {
   return undefined;
 };
 
+const getUsers = (state: RootState): User[] | undefined => {
+  const users = selectUsers(state);
+
+  if (users.length === 0) {
+    return undefined;
+  }
+
+  return users.map((user) => {
+    const result: User = {
+      name: user.name,
+    };
+    if (user.password !== undefined) {
+      result.password = user.password;
+    }
+    if (user.ssh_key !== undefined) {
+      result.ssh_key = user.ssh_key;
+    }
+    if (user.groups !== undefined) {
```

Comment:
Same as the comment above here. However, since we don't support groups yet, let's just remove this for now and save it for the isAdmin pr.